app/controllers/my_notifications.js

```diff
@@ -32,25 +30,30 @@ export default Ember.Controller.extend({
   onNewNotification(notification) {
     const store = this.get("store");
     const msg = store.peekRecord("message", notification.record.id);
-    const orderId = notification.record.order_id;
+    const messageableId = notification.record.messageable_id;
     const notifications = this.get("notifications");
 
-    if (!orderId || this.session.router.currentURL !== "/my_notifications") {
+    if (
+      !messageableId ||
+      this.session.router.currentURL !== "/my_notifications"
```

Comment:
Since this is the `my_notifications` controller, would the currentURL ever be something other than `/my_notifications` ?